daprdocs/content/en/developing-applications/building-blocks/actors/namespaced-actors.md

```diff
@@ -0,0 +1,109 @@
+---
+type: docs
+title: "Namespaced actors"
+linkTitle: "Namespaced actors"
+weight: 40
+description: "Learn about namespaced actors"
+---
+
+Namespaced actors use the multi-tenant Placement service. With this service, in a scenario where each tenant has its own namespace, sidecars belonging to a tenant named "Tenant A" won't receive placement information for "Tenant B". 
+
+Namespacing in Dapr provides isolation, and thus multi-tenancy. With namespacing, the same actor type can be deployed into different namespaces. Instances of actors can call across these namespaces. 
+
+{{% alert title="Note" color="primary" %}}
+Note that the namespace is not part of the actor name (that is an actor id), namespace is used for the isolation of the deployed actor services.
+{{% /alert %}}
+
```

Comment:
{{% alert title="Note" color="primary" %}}
Note that each namespaced actor deployment must use its own separate state store, especially if the same actor type is used across namespaces. In other words, no namespace information is written as part of the actor record, and hence separate state stores are required for each namespace. See "Configuring actor state stores for namespacing" section below.
{{% /alert %}}

Put the proper link in for the "Configuring actor state stores for namespacing"